lib/src/property.dart

```diff
@@ -62,16 +62,16 @@ class Color implements _StyleProperty, ColorBase {
   /// '0' (completely transparent) to '1.0' (completely opaque).  It will
   /// internally be mapped to an int between '0' and '255' like the other color
   /// components.
-  Color.createRgba(int red, int green, int blue, [num alpha])
+  Color.createRgba(int red, int green, int blue, [num? alpha])
       : _argb = Color.convertToHexString(
-            Color._clamp(red, 0, 255),
-            Color._clamp(green, 0, 255),
-            Color._clamp(blue, 0, 255),
+            Color._clamp(red, 0, 255) as int,
```

Comment:
I would consider making `Color._clamp` generic to avoid all these casts here and below.

Reply:
Good call. Done.